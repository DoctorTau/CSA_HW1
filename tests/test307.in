18
-6335 -9325 -1752 206 6241 -4086 6866 5172 6445 -5454 -5088 -8406 -8966 3338 5896 5802 -8007 7126 