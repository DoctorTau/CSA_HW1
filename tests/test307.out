-1752 -4086 -5454 -5088 -8406 -8966 